src/libraries/System.Runtime/tests/System/Diagnostics/CodeAnalysis/RequiresDynamicCodeAttributeTests.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Doesn't this file need to be added to https://github.com/dotnet/runtime/blob/fb53b5c535866f16f310ea05096a2b9f102c2756/src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj?

Reply:
I was looking at the files we included for RAF and apparently that was missing too, I added both files to the csproj